suite-native/module-accounts-import/src/accountsImportThunks.ts

```diff
@@ -1,24 +1,28 @@
 import { createThunk } from '@suite-common/redux-utils';
-import { accountsActions, selectAccounts } from '@suite-common/wallet-core';
+import { accountsActions, selectDeviceNetworkAccounts } from '@suite-common/wallet-core';
 import { AccountInfo } from '@trezor/connect';
 import { NetworkSymbol } from '@suite-common/wallet-config';
 import { createDevice, selectDeviceById } from '@suite-native/module-devices';
 
-const actionPrefix = '@accountImport';
+const actionPrefix = '@accountsImport';
 
 type ImportAssetThunkPayload = {
     deviceId: string;
     deviceTitle: string;
     accountInfo: AccountInfo;
+    accountLabel: string;
     coin: NetworkSymbol;
 };
 
-const getMockedDeviceState = (deviceId: string) => `state@${deviceId}:1`;
+// This is temporary solution for MVP purposes. We have one hidden device and all imported accounts belongs to it.
+export const HIDDEN_DEVICE_ID = 'hiddenDeviceWithImportedAccounts';
+
+export const getMockedDeviceState = (deviceId: string) => `state@${deviceId}:1`;
```

Comment:
Why do we need this function? Why not to use just `const DEVICE_STATE = 'state@${HIDDEN_DEVICE_ID}:1'`?

Reply:
We don't really need it now. It was a leftover from the previous solution with more devices... Fixed.